qiskit/compiler/transpile.py

```diff
@@ -303,13 +321,33 @@ def _transpile_circuit(circuit_config_tuple: Tuple[QuantumCircuit, Dict]) -> Qua
     if ms_basis_swap is not None:
         pass_manager.append(MSBasisDecomposer(ms_basis_swap))
 
+    # Should scheduling incorporate with preset pass managers?
+    scheduling_method = pass_manager_config.scheduling_method
+    if scheduling_method is not None:
```

Comment:
Right now, if I don't pass `scheduling_method`, and I try to use the transpiler as it was, I get an error (`TranspilerError: 'DurationMapper needs backend.configuration().dt'`).

This is because `scheduling_method = [None]` so this evals to True

Reply:
Fixed at b58098b